com.tibia.Tibia.yml

```diff
@@ -0,0 +1,63 @@
+id: com.tibia.Tibia
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: tibia-wrapper
+tags:
+  - proprietary
+
+finish-args:
+  - --device=dri # OpenGL rendering
+  - --filesystem=xdg-data/tibia:create
```

Comment:
why is xdg-data needed ?

Reply:
The game stores configuration in this folder. Should I comment/document it?

Reply:
XDG_DATA_HOME inside Flatpak is `~/.var/app/$FLATPAK_ID/data`. Why does it need host's xdg-data which is usually `~/.local/share` ?

Reply:
Indeed it respects `XDG_DATA_HOME`, will remove it